fastify-session-authentication/authentication.test.js

```diff
@@ -0,0 +1,116 @@
+'use strict'
+
+const { test } = require('tap')
+const server = require('./authentication')
+const requestCallback = require('request')
+
+server.then((app) => {
+  app.server.unref()
```

Comment:
We should instantiate one server for each test, ideally in a `beforeEach`.

Reply:
I instantiate one server for each test now.